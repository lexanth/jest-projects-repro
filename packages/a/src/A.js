import React from 'react'
import { C } from '@jest-repro/c'

export const A = () => (
  <div data-testid="A">
    <C />
  </div>
)
