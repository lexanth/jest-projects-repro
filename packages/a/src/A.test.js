import { render } from '@testing-library/react'
import { A } from './A'
import React from 'react'

describe('A', () => {
  it('should render', () => {
    const { container } = render(<A />)
    expect(container).toMatchSnapshot()
  })
})
