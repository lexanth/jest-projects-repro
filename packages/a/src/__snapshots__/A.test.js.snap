// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A should render 1`] = `
<div>
  <div
    data-testid="A"
  >
    <div
      data-testid="C"
    />
  </div>
</div>
`;
